testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -4777,6 +4777,23 @@ QUnit.module('keyboard navigation', {
         assert.ok($cancelButton.hasClass('dx-state-focused'), 'cancel button is focused');
     });
 
+    QUnit.testInActiveWindow('Popup has open state if tab key was pressed when applyValueMode: "instantly"', function(assert) {
+        if(devices.real().deviceType !== 'desktop') {
```

Comment:
Look's like we unnecessary to add this condition, seems there is no specific in this case. May be I missed something?

Reply:
Hey, thank you for your attention! Fixed in [99a178d2a43ec5be8ad8dd319a440caaa9118077].